Sources/UInt128.swift

```diff
@@ -398,26 +398,7 @@ extension UInt128 : BinaryInteger {
     // MARK: Instance Methods
 
     public var words: [UInt] {
-        guard self != UInt128.min else {
-            return []
-        }
-
-        var words: [UInt] = []
-
-        for currentWord in 0 ..< self.bitWidth / UInt.bitWidth {
-            let shiftAmount: UInt64 = UInt64(UInt.bitWidth * currentWord)
-            let mask = UInt64(UInt.max)
-            var shifted = self
-
-            if shiftAmount > 0 {
-                shifted &>>= UInt128(upperBits: 0, lowerBits: shiftAmount)
-            }
-
-            let masked: UInt128 = shifted & UInt128(upperBits: 0, lowerBits: mask)
-
-            words.append(UInt(masked.value.lowerBits))
-        }
-        return words
+        return Array(value.lowerBits.words) + Array(value.upperBits.words)
```

Comment:
i wanted to put this into a separate pr but i fucked up lol. anyway, i think this is more correct than the previous implementation as for example UInt64.min.words doesn't return []

Reply:
Yeah, that change looks slick. Do the tests still pass, or do we need to change the tests to handle the min case?

Also, have you looked at my other comments?

Reply:
yeah, tests pass. 

i have yeah, sorry i added the other fixes after this. i still dont have a good workflow for like pushing a pr and then going back and redoing things if i have already committed other stuff.

Reply:
The easiest way is to create separate branches for each PR. Rebase branches against approved PRs so that you can keep them in a consistent state, at least when the branches all deal with the same file being modified.